dune-project

```diff
@@ -32,4 +32,5 @@ Based on Modal Type Theory as given by F. Pfenning et al.")
   sedlex
   sexplib
   stdio
+  qcheck
```

Comment:
Let's move `qcheck` right after `ppx_string_interpolation`, the dependency list is sorted alphabetically.